### DISCUSSION TITLE
Which installation methods are recommended, and is the PyPi package equivalent?

### DISCUSSION THREAD

**Discussion by dmyersturnbull** [boisgera/pandoc#73] [Q&A]
[pandoc.org/installing.html](https://pandoc.org/installing.html) doesn't mention the [pandoc PyPi package](https://pypi.org/project/pandoc/). Does that package have a limitation or is otherwise not recommended? Thanks!

**Comment by boisgera** [comment]
Hi!

You always need "Pandoc-the-CLI-tool" (from https://github.com/jgm/pandoc), which is **not** included in "Pandoc-the-Python-library" (this project, packaged for PyPi).

Do you need to make some advanced document transforms, handled by a Python program? Otherwise you may not even require our project but only the CLI tool.




Cheers,

Sébastien

**Comment by dmyersturnbull** [comment]
@boisgera Thanks!
